__tests__/app.test.js

```diff
@@ -90,3 +91,58 @@ describe("Err /api/articles/article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("200: responds with an articles array of article objects and a new comment_count property", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        body.forEach((column) => {
```

Comment:
It would be beneficial to check the length of the array here to make sure all of the articles you expect are being returned